src/display-layer.coffee

```diff
@@ -62,7 +62,6 @@ class DisplayLayer
 
   constructor: (@id, @buffer, settings={}) ->
     @displayMarkerLayersById = {}
-    @textDecorationLayer = null
```

Comment:
It should be okay to 🔥 this because we are assigning that ivar to an `EmptyDecorationLayer` on line 75.